src/HeaderBar.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import './HeaderBar.scss';
+
+class HeaderBar extends Component {
+  render() {
+    return (
+      <header className='mdc-top-app-bar'>
+        <div className='mdc-top-app-bar__row'>
+          <section className='mdc-top-app-bar__section mdc-top-app-bar__section--align-start'>
+            <a href='/' className='material-icons mdc-top-app-bar__navigation-icon'>
+              <img src='ic_component_24px_white.svg'></img>
```

Comment:
`<img>` is a self-closing tag, so we can drop the `</img>`.

We should probably add some kind of `alt` text for the image, and/or a `title` attribute on the `<a>`.

I just noticed there's even a console warning about it in the dev tools:

> Line 11:  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text